iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/IcebergTableOpsManager.java

```diff
@@ -97,8 +126,20 @@ private String shelling(String rawPrefix) {
     }
   }
 
+  private void closeIcebergTableOps(IcebergTableOps ops) {
+    try {
+      ops.close();
+    } catch (Exception ex) {
+      LOG.warn("close iceberg table ops fail: {}", ops);
+      throw new RuntimeException(ex);
```

Comment:
what will happen if throwing exception here? I prefer to just print warning logs.

Reply:
ok